pkg/config/config.go

```diff
@@ -37,6 +38,28 @@ type Config struct {
 
 // Validate does static validation of the configuration
 func (c Config) Validate() error {
+	// Validate kernel version
+	// For valid versions see https://aquasecurity.github.io/tracee/latest/getting-started/installing/prerequisites/
+
+	// Default minimum kernel version
+	minimalKernelVersion := "5.4"
+	if c.OSInfo.GetOSReleaseID() == helpers.CENTOS {
```

Comment:
not only centos but also rhel
Actually, 4.19 is the minimal version now and 5.4 is required for the network features only

Reply:
I went by the prerequisite given in the tracee manual. I see that our BTFs don't align with this check. It could be the case that this PR may be too premature, as we need a better image of the validation.